test/setCommand_test.js

```diff
@@ -18,48 +18,48 @@ describe('The set command', () => {
   beforeEach(() => {
     dataLayer = [];
     dlh = new DataLayerHelper(dataLayer);
-    commandAPI('set', 'bar', 'foo');
-    targetModel = Object.assign({}, dlh.abstractModelInterface_);
+    targetModel = {};
   });
 
+
   describe('with 2 argument format', () => {
     it('updates model without corrupting it.', () => {
       commandAPI('set', 'foo', 'bar');
       targetModel['foo'] = 'bar';
 
-      expect(dlh.abstractModelInterface_).toEqual(targetModel);
+      expect(dlh.model_).toEqual(targetModel);
```

Comment:
Good catch. Should we also check the abstractModelInterface_ though?

Reply:
Not necessarily in all tests.

Reply:
I added a test to make sure we don't modify it, I don't think we need to test it any more than that since the set command is somewhat unrelated to the abstract model interface.